ui-v2/app/styles/components/with-tooltip/index.scss

```diff
@@ -1,2 +1,25 @@
 @import './skin';
 @import './layout';
+%with-pseudo-tooltip {
+  @extend %tooltip;
+}
+%with-pseudo-tooltip::before {
+  @extend %tooltip-bubble;
+}
+%with-pseudo-tooltip::after {
+  @extend %tooltip-tail;
+}
+%with-pseudo-tooltip {
+  text-indent: -9000px;
+  font-size: 0;
+  top: -9px;
+}
+
+%with-pseudo-tooltip::after,
+%with-pseudo-tooltip::before {
+  @extend %blink-in-fade-out;
+}
+%with-pseudo-tooltip:hover::after,
+%with-pseudo-tooltip:hover::before {
+  @extend %blink-in-fade-out-active;
+}
```

Comment:
I'm unsure whether to move this somewhere else or not. Its basically a 'helper' placeholder. So it's not skin or layout, its just a helper so if you want a tooltip using pseudo content its really easy to do via:

```css
[data-tooltip] {
  @extend %with-pseudo-tooltip;
}
```
I'd say it belongs with the component files, but possibly in a different file to this one? Possibly not?